speechmatics/cli.py

```diff
@@ -230,6 +230,11 @@ def get_transcription_config(
     ]:
         config[option] = True if args.get(option) else config.get(option)
 
+    if args.get("volume_threshold"):
+        config["audio_filtering_config"] = {
```

Comment:
can't this be a property in the top level dict?

Reply:
Do you mean `audio_filtering_config` should be a top level property of the whole thing, or that `volume_threshold` should be a property in `transcription_config`?

If the first, then I think audio filtering is a transcription option (and unlike a standalone option configuring a speech capability)

If the second, it's because it follows the related openapi-spec MR and `audio_filtering_config` is open for extension, even though it only has one setting at the moment - the analogy is with ``speaker_diarization_config`.

Reply:
Resolving since the openapi-specifications change is merged.